inc/apirest.class.php

```diff
@@ -189,6 +189,13 @@ public function call() {
 
          return $this->returnResponse($response, $code, $additionalheaders);
 
+      } else if ($resource === "lostPassword") {
+         if ($this->verb != 'POST' && $this->verb != 'PUT') {
+            // forbid password reset with GET because it may be logged
+            return $this->returnError();
```

Comment:
maybe precise that this endpoint couldn't be used by GET.
By the way, you have also an inconsistence with the inline documentation which precise GET verb for this endpoint